src/InsightsBrakes.js

```diff
@@ -0,0 +1,90 @@
+import { Carousel } from 'antd';
+import './insights-styles.css'
+import React from 'react';
+
+const contentStyle = {
+  height: '160px',
+  color: '#fff',
+  lineHeight: '160px',
+  textAlign: 'center',
+  background: '#364d79',
+};
+
+const App = () => (
+  <Carousel autoplay>
+
+    <div style={contentStyle}>
+        <section>
+            <h1> Ceramic Brake Pads</h1>

```

Comment:
try implementing bootstrapping tags to help both with documentation and clearly group related code